core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -123,7 +123,10 @@ public void setup() {
       new DateFormatTester("2008-12-31",
           ImmutableList.of("%v","%V","%u","%U"),
           ImmutableList.of("53","52","53","52")
-      )
+      ),
+      new DateFormatTester("1998-01-31 13:14:15.012345",
```

Comment:
add not-match cases? e.g. "%Y-%m-%da%Ta"

Reply:
Added few more testcases.